habana/1.17.1/ubi9-python-3.11/Dockerfile

```diff
@@ -0,0 +1,231 @@
+# Copyright (c) 2024 Habana Labs, Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# HabanaLabs Dockerfile base installer layer for RedHat 8.6
+# Reference: https://github.com/HabanaAI/Setup_and_Install/blob/1.13.0/dockerfiles/base/Dockerfile.rhel8.6 
+ARG BASE_IMAGE
```

Comment:
This base image is what we define [here](https://github.com/opendatahub-io/notebooks/pull/695/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52) in that case jupyter-datascience.

I am not sure if we include somehow this [vault](https://github.com/opendatahub-io/notebooks/pull/533/files#diff-980ed0a5b280defea81d48bddada2d8f0eb90026a93b216b3651c44ddfdae432) base image from this example PR Or i am missing something?

Reply:
We are using a mix of their Dockerfile and our own. I will change the description to explain this briefly.